resources/views/mail/travel/assignmentcreated.blade.php

```diff
@@ -0,0 +1,23 @@
+Hi {{ $assignment->user->preferred_first_name }},
+
+@if($assignment->travel->tar_required)
+You have been assigned to {{ $assignment->travel->name }}. Please complete the following items as soon as possible so that we can book travel for you.
+
+Visit {{ route('docusign.travel') }} to submit a Travel Authority Request. Georgia Tech requires this form to be submitted for all official travel.
+
+Make a ${{ intval($assignment->travel->fee_amount) }} payment for the travel fee.
```

Comment:
This should probably move below the conditional.

Reply:
Do you mean inside the `@else` or after the `@endif`?

Reply:
After the `@endif`.

Reply:
The `@else` mentions the travel fee already.

Reply:
Oops. Disregard